### PR TITLE
Add Military rooftop sniper post

### PR DESCRIPTION
**Pull request by faefux**
#### Summary

```SUMMARY: Content "military rooftop observation/sniper post"```

#### Purpose of change
To start to make the cities look more militarized like they were in a state of martial law.

#### Describe the solution
Adding a military rooftop observation/sniper post that only spawns on the doctors office roof for now but can be added to other roofs depending on how common people think it should be.

#### Describe alternatives you've considered
Not adding

#### Additional context
This definitely could be added to more roofs but i'm not sure which and cant go through them right now.  This was a quick idea and implementation i had, so i'd really appreciate any changes or pointers.

### CODE REVIEW

File: data/json/mapgen/nested/nested_chunks_roof.json [hunk 1]
```diff
@@ -496,5 +496,31 @@
...
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rows": [
+        "xx~~",
+        "xo.~",
+        "_..x",
+        "__xx"
+      ],
+      "terrain": { "_": "t_concrete", "x": "t_concrete", "~": "t_concrete", ".": "t_concrete" },
+      "furniture": { "o": "f_tourist_table", "x": "f_sandbag_wall", "~": "f_sandbag_half", ".": "f_groundsheet" },
```
Comment (Night-Pryanik) [inline data/json/mapgen/nested/nested_chunks_roof.json]: `o` symbol has furniture set, but no underlying terrain.

**Comment by Night-Pryanik** [comment]
Please test your changes locally in your game first.

File: data/json/mapgen/nested/nested_chunks_roof.json (hunk 1 above)
Comment (faefux) [reply data/json/mapgen/nested/nested_chunks_roof.json]: Done. Thanks. :)

**Comment by faefux** [comment]
I do check them first. Why wouldn't I?

**Comment by Night-Pryanik** [comment]
A lot of errors should popup when trying to start the game, for example, with no terrain set for aforementioned `o` symbol. It's strange if they don't appear, or if you somehow didn't notice them.

**Comment by faefux** [comment]
I've managed to fuck up pasting from my machine to github somewhere along the line and it's 6am so thinking straight is hard. going to temporary clsoe this and come back to it when i get some sleep.

**Comment by faefux** [comment]
> A lot of errors should popup when trying to start the game, for example, with no terrain set for aforementioned `o` symbol. It's strange if they don't appear, or if you somehow didn't notice them.

I didn't get an error for that, odd! :s

**Pull request closed without merging**